modules/core/src/main/java/org/apache/synapse/rest/API.java

```diff
@@ -59,6 +60,7 @@
     private Map<String,Resource> resources = new LinkedHashMap<String,Resource>();
     private List<Handler> handlers = new ArrayList<Handler>();
     private String swaggerResourcePath;
+    private RESTUtils restUtils;
```

Comment:
Util classes are usually classes that hold static util methods. Is this initialization essential?

Reply:
Fixed